unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -741,6 +742,21 @@ public static UTF8String concatWs(UTF8String separator, UTF8String... inputs) {
     return res;
   }
 
+  // TODO: Need to use `Code Point` here instead of Char in case the character longer than 2 bytes
```

Comment:
We could have a fast version (using byte), and a slow version (using int as code point), that could be a separate PR.